install.sh

```diff
@@ -328,8 +334,11 @@ else
   # by default. For more on why this setup might be useful see:
   # https://github.blog/2020-01-17-bring-your-monorepo-down-to-size-with-sparse-checkout/
   # If using alternate branch don't use depth=1 .
-  [[ "$BRANCH" == "master" ]] && GIT_DEPTH="--depth=1" || GIT_DEPTH=
-  git clone "${GIT_DEPTH}" --filter=blob:none --no-checkout "https://github.com/${OWNER}/${REPO}.git" "${CREW_LIB_PATH}"
+  if [[ "$BRANCH" == "master" ]]; then
+    git clone --depth=1 --filter=blob:none --no-checkout "https://github.com/${OWNER}/${REPO}.git" "${CREW_LIB_PATH}"
+  else
+    git clone --filter=blob:none --no-checkout "https://github.com/${OWNER}/${REPO}.git" "${CREW_LIB_PATH}"
+  fi
```

Comment:
What was wrong with the previous logic?

Reply:
The variable got passed as `''` which for some reason was being seen as a variable, which was making git complain.

Changing the logic to this made it work. 🤷‍♂️